shardingsphere-governance/shardingsphere-governance-core/src/main/resources/META-INF/services/org.apache.shardingsphere.governance.core.registry.listener.GovernanceListenerFactory

```diff
@@ -20,3 +20,4 @@ org.apache.shardingsphere.governance.core.registry.listener.factory.DataSourceSt
 org.apache.shardingsphere.governance.core.registry.listener.factory.LockChangedListenerFactory
 org.apache.shardingsphere.governance.core.registry.listener.factory.PropertiesChangedListenerFactory
 org.apache.shardingsphere.governance.core.registry.listener.factory.MetaDataChangedListenerFactory
+org.apache.shardingsphere.governance.core.registry.listener.factory.GlobalRuleChangedListenerFactory
```

Comment:
Could you add UT for GlobalRuleChangedListenerFactor and other factories, you can fix them in next PR

Reply:
OK

Reply:
looking forward your next PR.